tests/functional/pages/surveys/census/communal/further-contact.page.js

```diff
@@ -1,4 +1,4 @@
-// >>> WARNING THIS PAGE WAS AUTO-GENERATED ON 2016-12-14 14:19:14.086947 - DO NOT EDIT!!! <<<
+// >>> WARNING THIS PAGE WAS AUTO-GENERATED - DO NOT EDIT!!! <<<
```

Comment:
On the flipside, it's also really annoying to review a PR where the change is muddled in amongst hundreds of trivial touches like this. Probably should have been the subject of its own PR.